data/json/recipes/recipe_ammo.json

```diff
@@ -951,7 +951,39 @@
       ]
   ]
 },{
-   "type" : "recipe",
+  "type" : "recipe",
+  "result": "bolt_steel",
+  "category": "CC_AMMO",
+  "subcategory": "CSC_AMMO_ARROWS",
+  "skill_used": "fabrication",
+  "skills_required": ["archery", 5],
+  "difficulty": 5,
+  "time": 10000,
+  "reversible": false,
+  "autolearn": true,
+  "book_learn" : [[ "recipe_arrows", 2 ]],
+  "qualities":[
+    {"id":"HAMMER","level":1,"amount":1}
+  ],
+     "tools": [
+    [
+      ["toolset", 7],
+      ["fire", -1],
+      ["forge", 7],
+      ["char_forge", 2],
+      ["oxy_torch", 2]
+      ]
+    ],
+     "components": [
+    [
+      ["steel_lump", 4]
```

Comment:
Steel lumps are large amounts of steel--largest "scrap" amount available.  I'm thinking chunks might be more suitable.